### PR TITLE
Add on textbox focus

### PR DESCRIPTION
**Pull request by Hona**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] Changelog is provided or not needed

### CODE REVIEW

**Comment by Hona** [comment]
Test please @ElderJames

File: components/input-number/InputNumber.razor.cs [hunk 1]
```diff
@@ -74,6 +74,9 @@ public TValue DefaultValue
         [Parameter]
         public EventCallback<TValue> OnChange { get; set; }
 
+        [Parameter]
+        public Action OnTextboxFocus { get; set; }
```
Comment (anddrzejb) [inline components/input-number/InputNumber.razor.cs:78]: Wouldn't it be better to use the same name as in `Input` component? I am talking about [`OnFocus`](https://github.com/ant-design-blazor/ant-design-blazor/blob/de2e6681dc594c92bbd4a3f793ee7b8e6bdbe8d0/components/input/Input.cs#L127)? And also `EventCallback<FocusEventArgs>` for the type.

Comment (ElderJames) [reply components/input-number/InputNumber.razor.cs:78]: I would like to have all input-type components add the focus method. Can this be done together?

Comment (Hona) [reply components/input-number/InputNumber.razor.cs]: @ElderJames Sure. Let me take a look at this later

**Review by ElderJames — APPROVED** [review]

**Pull request merged**